homeassistant/components/sensor/__init__.py

```diff
@@ -187,17 +184,16 @@ def state_attributes(self) -> dict[str, Any] | None:
                 report_issue = self._suggest_report_issue()
                 _LOGGER.warning(
                     "Entity %s (%s) with state_class %s has set last_reset. Setting "
-                    "last_reset for entities with state_class other than 'total' is "
-                    "deprecated and will be removed from Home Assistant Core 2021.10. "
-                    "Please update your configuration if state_class is manually "
-                    "configured, otherwise %s",
+                    "last_reset is deprecated and will be unsupported from Home "
+                    "Assistant Core 2021.11. Please update your configuration if "
+                    "state_class is manually configured, otherwise %s",
```

Comment:
```suggestion
                    "Assistant Core 2021.11. Please update your configuration and set state class to 'total_increasing' if "
                    "state_class is manually configured, otherwise %s",
```